app/api/server/v1/invites.js

```diff
@@ -39,8 +39,8 @@ API.v1.addRoute('validateInviteToken', { authRequired: false }, {
 			throw new Meteor.Error('error-invalid-token', 'The invite token is invalid.', { method: 'validateInviteToken', field: 'token' });
 		}
 
-		const result = validateInviteToken(token);
+		validateInviteToken(token);
```

Comment:
Here I'd not return an error if the token is invalid, would do a try catch to return true or false with the error name